printer_mods/oxno/rbp_zero_braket/README.md

```diff
@@ -0,0 +1,9 @@
+This is a bracket for a raspberry pi zero / w
+the bracket mounts to the din rail using "pcb_din_clip_x3.stl"
+
+To mount the rbp zero w, I used steel thread rolling screws for plastic pan head #3-24 thread size, 5/16" length.
+
+![rbp_bracket_front](https://user-images.githubusercontent.com/45470717/120405358-5fddcb80-c316-11eb-8182-c176a806990d.png)
```

Comment:
Please put these images into a directory with your mod, such as `printer_mods/oxno/rbp_zero_bracket/rbp_bracket_front.png`.

Reply:
The slot is for the camera ribbon cable if needed.

Reply:
Now that you put the images into this directory, can you update the README to reference them instead of the ones in githubusercontent ?  Something like

    ![rbp_bracket_front](rbp_bracket_front.png)

Also, you still need to add your mod to `printer_mods/README.md`.